.github/workflows/rust.yml

```diff
@@ -62,13 +62,6 @@ jobs:
         run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
       - run: wasm-pack build --target nodejs bindings/wasm
 
-  bench:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Bench
-        run: cargo bench
-
```

Comment:
You added quite a lot of benchmarks the past 2 weeks and this now takes 7 minutes or more, so I suggest to separate.

No need for Nyrkiö to do that. Just a suggestion in general.